test.lua

```diff
@@ -9,14 +9,16 @@ minetest.register_privilege("monoid_master", {
 
 local function test(player)
```

Comment:
Rather pass the player name here, and check for the player existence within this function and `minetest.after` as well. The player might leave during these 3 seconds.